src/node_buffer.cc

```diff
@@ -169,7 +169,8 @@ void CallbackInfo::WeakCallback(Isolate* isolate) {
 // Parse index for external array data.
 inline MUST_USE_RESULT bool ParseArrayIndex(Local<Value> arg,
                                             size_t def,
-                                            size_t* ret) {
+                                            size_t* ret,
+                                            size_t needed = 0) {
```

Comment:
Yeah, I mean, inside of `ParseArrayIndex`? I feel like I am missing the point of passing this argument…